l10n_nl_intrastat/models/l10n_nl_intrastat.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning
```

Comment:
adapt this to the 6th bullet in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms